jubilant/_juju.py

```diff
@@ -2,9 +2,13 @@
 import logging
 import os
 import subprocess
+import tempfile
 import time
-from collections.abc import Callable, Iterable
+from collections.abc import Callable, Iterable, Mapping
+from typing import Any
 
+from . import ActionError, _yaml
+from ._actions import ActionResult
```

Comment:
This feels a little weird. Wouldn't it be cleaner to import `ActionError` and `ActionResult` from the same place? And that place be `._actions`? I don't love a `from .` that imports a class and a module.

Reply:
Yep, good call -- just an oversight. Fixed.